src/init.cpp

```diff
@@ -1757,7 +1757,7 @@ bool AppInitMain()
 
 
     // Automint denom
-    if(nPreferredDenom == DEFAULT_AUTOMINT_DENOM) {
+    if(gArgs.IsArgSet("-nautomintdenom")){
         nPreferredDenom = gArgs.GetArg("-nautomintdenom", DEFAULT_AUTOMINT_DENOM);
         if(nPreferredDenom != 10 && nPreferredDenom != 100 && nPreferredDenom != 1000 && nPreferredDenom != 10000){
             nPreferredDenom = DEFAULT_AUTOMINT_DENOM;
```

Comment:
There should be a log message (at least) informing that the nautomintdenom is invalid and using the default instead (similar to other init messages that are logged).  Have to tell the user somehow that they need to go fix it.  Since the UI setting isn't the preference anymore, this now becomes more important; as there would be no way to understand that their nautomintdenom=20000 is actually using 1000 [for example]

Reply:
Sure.  This was pre-existing without any log message but I will certainly make that change for clarity